help/de/Help_ch_Account-Actions.xml

```diff
@@ -829,61 +839,103 @@
+                    heruntergeladen werden können.
+                  </para>
+                </listitem>
+              </varlistentry>
 
-                <para>Nachdem Sie die Art der Quelle für Kursangebote ausgewählt haben, wählen Sie eine Datenquelle aus
-                  dem Pulldown-Menü. Details zu den derzeit unterstützten Datenquellen finden Sie
-                  unter <xref linkend="fq-sources"></xref>.
-                </para>
+              <varlistentry>
+                <term><guilabel>Art der Kursabruf-Quelle</guilabel></term>
 
-                <note>
-                  <para>Die Kursdatenquelle "Yahoo" liefert dabei auch die Kurse für die übrigen genannten Unternehmen
-                    (z.B. Fidelity Investments, T.Rowe Price und The Vanguard Group) und gibt den
-                    Kurs normalerweise identisch zur Original-Datenquelle an.
+                <listitem>
+                  <para>Verwenden Sie die Optionsfelder, um die Quelle für die Online-Kurse auszuwählen.
                   </para>
-                </note>
 
-                <para>Beachten Sie allerdings, dass Sie bei <quote>Yahoo Europe</quote> zusätzlich das
-                  Börsenplatzkürzel des Wertpapiers anfügen sollten, z.B. <quote>PA</quote> für
-                  Paris, <quote>BE</quote> für Berlin usw. Beispiel: 12150.PA (ein Peugeot-Papier
-                  am Pariser Markt). <xref linkend="gnc-tbl-fq-yahoo-exchange-codes"></xref>
-                  enthält eine Liste von Suffixen für verschiedene Märkte auf der ganzen Welt.
-                </para>
+                  <variablelist spacing="compact">
+                    <varlistentry>
+                      <term><guilabel>Einzel</guilabel></term>
+
+                      <listitem>
+                        <para>Diese Auswahl ruft die Daten nur aus der angegebenen Quelle ab.
+                        </para>
+                      </listitem>
+                    </varlistentry>
+
+                    <varlistentry>
+                      <term><guilabel>Mehrere Kursquellen</guilabel></term>
+
+                      <listitem>
+                        <para>Kursquellen wie <quote>Europa</quote> sollten verwendet werden, wenn eine Ausfallsicherung
+                          gegenüber mehreren Datenquellen erwünscht ist.
+                        </para>
+
+                        <caution>
+                          <para>Die Option <guilabel>Mehrere Kursquellen</guilabel> ist nur sinnvoll, wenn alle ausgewählten
+                            Quellen das gleiche Symbol für Ihre Wertpapier verwenden.
+                          </para>
+                        </caution>
+                      </listitem>
+                    </varlistentry>
+
+                    <varlistentry>
+                      <term><guilabel>unbekannt</guilabel></term>
+<!-- FIXME source: What function does this option have? -->
```

Comment:
Wenn ich mal Zeit habe,pflege ich den aktuellen Stand aus dem F::Q-Quelltext in 

1. https://github.com/Gnucash/gnucash/blob/maint/libgnucash/engine/gnc-commodity.c
2. https://github.com/Gnucash/gnucash-docs/blob/maint/help/C/Help_tips-appendix.xml etc. ein

Wenn die Quelle nicht in gnc-commodity.c definiert ist, ist sie eben unbekannt, Dafür gibt es dann 2 mögliche Gründe:

1. Ich hänge aktuell eine Version hinterher oder
2. Der Benutzer hängt hinterher und hat z.B. ZürichInvest noch nicht auf DWS-Fonds geändert.

Reply:
Leider habe ich keine Idee wie ich das formulieren könnte :-(

Reply:
```
&app; hat hier kein Hintergrundwissen über die Quelle. Das kann zwei verschiedene Gründe haben. Sie versuchen eine Quelle zu verwenden, die<itemizedlist>
<listitem><para>es nicht mehr in &app-fq; gibt oder</para></listitem>
<listitem><para>neu in &app-fq; ist, aber noch nicht in &app;.eingepflegt wurde.</para></listitem></itemizedlist>
Überprüfen Sie, ob <command>gnc-fq-check</command> Ihre Quelle listet und schlagen Sie in <xref appendixa…> die Umsetzung der Namen nach.

<!-- Maintainers: To add, modify, remove sources
edit https://github.com/Gnucash/gnucash/blob/maint/libgnucash/engine/gnc-commodity.c
and update the appendix. -->
```
oder so ähnlich

Reply:
ok.

Reply:
ok.